docker/Dockerfile

```diff
@@ -1,47 +1,48 @@
 # for optimized python builds
 # we follow solution #2 described in https://pythonspeed.com/articles/multi-stage-docker-python/
 
-FROM python:3 as builder
+FROM python:3.10-slim as builder
 
 RUN python -m venv /root/.venv
 
 ENV PATH="/root/.venv/bin:$PATH"
 
-ARG LNDMANAGE_HOST_SRC_PATH
+ARG LNDMANAGE_HOST_SRC_PATH .
```

Comment:
The dot seems not to be needed (got a syntax error), same below? https://docs.docker.com/engine/reference/builder/#arg

Reply:
Maybe because of differing docker versions that I don't get an error here; either way, just pushed a fix with the correct syntax.